nifi-nar-bundles/nifi-extension-utils/nifi-dbcp-base/src/main/java/org/apache/nifi/dbcp/utils/DefaultDataSourceValues.java

```diff
@@ -31,35 +29,28 @@ public Long getLongValue() {
         }
     },
     MAX_TOTAL_CONNECTIONS("8"),
+
     /**
      * Copied from {@link GenericObjectPoolConfig#DEFAULT_MIN_IDLE} in Commons-DBCP 2.7.0
      */
     MIN_IDLE("0"),
+
     /**
      * Copied from {@link GenericObjectPoolConfig#DEFAULT_MAX_IDLE} in Commons-DBCP 2.7.0
      */
     MAX_IDLE("8"),
-    /**
```

Comment:
I don't know why they were copied. I tried to avoid changing any logic that I wasn't 100% confident about. The ones that were removed, I believe, referenced deprecated methods. I used IntelliJ to just find any and all deprecated references and went through them 1-by-1, addressing. I made some other trivial updates that IntelliJ recommended as well, but largely didn't change anything else.